app/src/main/java/de/danoeh/antennapod/fragment/PlaybackHistoryFragment.java

```diff
@@ -267,4 +136,22 @@ public void onCreateContextMenu(ContextMenu menu, View v, ContextMenu.ContextMen
             MenuItemUtils.setOnClickListeners(menu, PlaybackHistoryFragment.this::onContextItemSelected);
         }
     }
+
+    @Override
+    protected List<FeedItem> loadData() {
+        return loadMoreData(0);
+    }
+
+    @Override
+    protected List<FeedItem> loadMoreData(int page) {
+        List<FeedItem> history = DBReader.getPlaybackHistory((page - 1) * EPISODES_PER_PAGE, EPISODES_PER_PAGE);
+        DBReader.loadAdditionalFeedItemListData(history);
```

Comment:
This is already called in `getPlaybackHistory`.

Reply:
OK, removed this.